types/networks/src/lib.rs

```diff
@@ -74,6 +74,10 @@ const VERSION_SCHEDULE: [Upgrade; 10] = [
         height: UPGRADE_ACTORS_V3_HEIGHT,
         network: NetworkVersion::V10,
     },
+    Upgrade {
+        height: UPGRADE_NORWEGIAN_HEIGHT,
+        network: NetworkVersion::V11,
+    },
```

Comment:
Wouldn't this make sense to order from most recent to least recent? That way it'd have to do fewer checks in the iterator? How often is `get_network_version_default` called?

Reply:
Oh cool, I missed that.